coffeetag/templates/stats.html

```diff
@@ -0,0 +1,144 @@
+<html lang="en">
+    <head>
+        <link rel="stylesheet" href="{{ url_for('static', filename='bootstrap-4.6.0-dist/css/bootstrap.min.css') }}">
+        <link rel="stylesheet" href="{{ url_for('static', filename='fontawesome-free-5.15.2-web/css/all.css') }}">
+        <link rel="stylesheet" href="{{ url_for('static', filename='coffeetag.css') }}">
+        <script src="{{ url_for('static', filename='jquery-3.6.0.min.js') }}"></script>
+        <script src="{{ url_for('static', filename='bootstrap-4.6.0-dist/js/bootstrap.bundle.min.js') }}"></script>
+        <script src="{{ url_for('static', filename='plotly-1.58.4.js') }}"></script>
+        <script>
+            $(() => {
```

Comment:
Do you think we need autologout for stats?

Reply:
I think we should have a timeout to all pages where the user is logged in. We might want to outsource the code to a .js file, as suggested in #11.

For stats, I set it to 60s (instead of 20s).